cep-incremental-repodata.md

```diff
@@ -0,0 +1,289 @@
+{"url": "https://conda.anaconda.org/conda-forge/linux-64/repodata.json", "latest": "ea8a695e21baa8090691656bbe4a4d2240ee62cdc2539dc5940077ccf2a44efc", "headers": {"date": "Tue, 26 Apr 2022 18:14:12 GMT", "content-type": "application/json", "transfer-encoding": "chunked", "connection": "keep-alive", "cf-ray": "70214bf9bd722293-MIA", "age": "670", "cache-control": "public, max-age=1200", "etag": "W/\"30e1be6bf1b73889a41eaa4846a59c56-23\"", "expires": "Tue, 26 Apr 2022 18:34:12 GMT", "last-modified": "Tue, 26 Apr 2022 17:38:23 GMT", "vary": "Accept-Encoding", "cf-cache-status": "HIT", "expect-ct": "max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"", "x-amz-id-2": "G2rn+4SD+sHXNxisSHtEbxHLulnvMh/Bj+KotOHzI0zxZDval2V8My839uHEfIC7LjtQRcnTOFA=", "x-amz-request-id": "7X74ZXZB2DMZA92F", "x-amz-version-id": "null", "set-cookie": "__cf_bm=.XMuRKJKwivDP9dELOOpA8NcS__lncLVi06qA6hKELM-1650996852-0-AaceO99Yc9N2lSPDz67hzvxp+BIrIYJE9Wj7HrfdzxL1xiTCx3fG6Cj/d0kODkOelFlXUp+eRnJ0uZE/oLxFysUZVVIu4GUx7gtisLdg5L6i; path=/; expires=Tue, 26-Apr-22 18:44:12 GMT; domain=.anaconda.org; HttpOnly; Secure; SameSite=None", "server": "cloudflare", "content-encoding": "gzip"}}
```

Comment:
This would be easier for me to read for the first time if I didn't have to scroll to the right and try to mentally parse what's happening.  I would show this json beautified but add a comment elsewhere explaining that each update is listed on a line by itself, but is shown here expanded for clarity.

```suggestion
{
  "to": "ea8a695e21baa8090691656bbe4a4d2240ee62cdc2539dc5940077ccf2a44efc",
  "from": "fda691c5ff4e265b580b0dc43cbb0bd95af9920a10bacf621b51e5cfd72baa03",
  "patch": [
    {
      "op": "add",
      "path": "/packages/mapbox_earcut-1.0.0-py310hbf28c38_2.tar.bz2",
      "value": {
        "build": "py310hbf28c38_2",
        "build_number": 2,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "numpy",
          "python >=3.10,<3.11.0a0",
          "python_abi 3.10.* *_cp310"
        ],
        "license": "ISC AND BSD-3-Clause",
        "md5": "93fc1b1a1e34d2b21fa2136d06aa6c89",
        "name": "mapbox_earcut",
        "sha256": "acaca6408c049b0575666614aa9a50f628f345a0b3b7d36ace8d55b2f005ff4d",
        "size": 80917,
        "subdir": "linux-64",
        "timestamp": 1650989008805,
        "version": "1.0.0"
      }
    },
    {
      "op": "add",
      "path": "/packages/mapbox_earcut-1.0.0-py37h506a98e_2.tar.bz2",
      "value": {
        "build": "py37h506a98e_2",
        "build_number": 2,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "numpy",
          "pypy3.7 >=7.3.7",
          "python >=3.7,<3.8.0a0",
          "python_abi 3.7 *_pypy37_pp73"
        ],
        "license": "ISC AND BSD-3-Clause",
        "md5": "b486d2bf0b9cb6f9fc9caf29aced70cf",
        "name": "mapbox_earcut",
        "sha256": "50242c2aff0fa007135e18b32c540b57460c53ca225f3ee182c13d1e13c7ed09",
        "size": 80007,
        "subdir": "linux-64",
        "timestamp": 1650988961306,
        "version": "1.0.0"
      }
    },
    {
      "op": "add",
      "path": "/packages/mapbox_earcut-1.0.0-py37h7cecad7_2.tar.bz2",
      "value": {
        "build": "py37h7cecad7_2",
        "build_number": 2,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "numpy",
          "python >=3.7,<3.8.0a0",
          "python_abi 3.7.* *_cp37m"
        ],
        "license": "ISC AND BSD-3-Clause",
        "md5": "a6227836fd9dc731a61ac0ed57589790",
        "name": "mapbox_earcut",
        "sha256": "050b3d269a0964c8bd814b2bdcf9f5c523225c3f602a18e4c04e1049438d4826",
        "size": 79206,
        "subdir": "linux-64",
        "timestamp": 1650988985131,
        "version": "1.0.0"
      }
    },
    {
      "op": "add",
      "path": "/packages/mapbox_earcut-1.0.0-py38h43d8883_2.tar.bz2",
      "value": {
        "build": "py38h43d8883_2",
        "build_number": 2,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "numpy",
          "python >=3.8,<3.9.0a0",
          "python_abi 3.8.* *_cp38"
        ],
        "license": "ISC AND BSD-3-Clause",
        "md5": "26ff06f1842618cd17ab7c99dc7a8a95",
        "name": "mapbox_earcut",
        "sha256": "d5e81828bf1cdc6e2105362cf867cb217d7fa38519628936d1a254b0be154bfc",
        "size": 80431,
        "subdir": "linux-64",
        "timestamp": 1650988985479,
        "version": "1.0.0"
      }
    },
    {
      "op": "add",
      "path": "/packages/mapbox_earcut-1.0.0-py39hf939315_2.tar.bz2",
      "value": {
        "build": "py39hf939315_2",
        "build_number": 2,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "numpy",
          "python >=3.9,<3.10.0a0",
          "python_abi 3.9.* *_cp39"
        ],
        "license": "ISC AND BSD-3-Clause",
        "md5": "4770c3dfb3b34f390e57d1fa3cf208f3",
        "name": "mapbox_earcut",
        "sha256": "6941e8d18d283b35e84596014c7ef8bca317ec320ae765eded8ad67fe9d9d0c7",
        "size": 80689,
        "subdir": "linux-64",
        "timestamp": 1650989007616,
        "version": "1.0.0"
      }
    },
    {
      "op": "add",
      "path": "/packages/poco-1.11.2-h08a2579_0.tar.bz2",
      "value": {
        "build": "h08a2579_0",
        "build_number": 0,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "openssl >=3.0.2,<4.0a0",
          "unixodbc >=2.3.9,<2.4.0a0"
        ],
        "license": "BSL-1.0",
        "license_family": "OTHER",
        "md5": "e1a33760d40c42bb21f10ebbf79fbd24",
        "name": "poco",
        "sha256": "6525ab8d6089481938e298dc50f8ca79dbd39ad3e616cbbd3daa9a9fb2cec295",
        "size": 5332156,
        "subdir": "linux-64",
        "timestamp": 1650989286660,
        "version": "1.11.2"
      }
    },
    {
      "op": "add",
      "path": "/packages/poco-1.11.2-h3790be6_0.tar.bz2",
      "value": {
        "build": "h3790be6_0",
        "build_number": 0,
        "depends": [
          "libgcc-ng >=10.3.0",
          "libstdcxx-ng >=10.3.0",
          "openssl >=1.1.1n,<1.1.2a",
          "unixodbc >=2.3.9,<2.4.0a0"
        ],
        "license": "BSL-1.0",
        "license_family": "OTHER",
        "md5": "5af5066afe0a90be9a2f95d92f3b225e",
        "name": "poco",
        "sha256": "bc86251c0a0f86146423996b8b8db30358a61d7f911c2322fe70ca2df7d752a8",
        "size": 5225400,
        "subdir": "linux-64",
        "timestamp": 1650989278591,
        "version": "1.11.2"
      }
    },
    {
      "op": "add",
      "path": "/packages/poppler-22.04.0-h1434ded_0.tar.bz2",
      "value": {
        "build": "h1434ded_0",
        "build_number": 0,
        "depends": [
          "boost-cpp >=1.74.0,<1.74.1.0a0",
          "cairo >=1.16.0,<1.17.0a0",
          "fontconfig >=2.13.96,<3.0a0",
          "fonts-conda-ecosystem",
          "freetype >=2.10.4,<3.0a0",
          "gettext >=0.19.8.1,<1.0a0",
          "jpeg >=9e,<10a",
          "lcms2 >=2.12,<3.0a0",
          "libcurl >=7.82.0,<8.0a0",
          "libgcc-ng >=10.3.0",
          "libglib >=2.70.2,<3.0a0",
          "libiconv >=1.16,<1.17.0a0",
          "libpng >=1.6.37,<1.7.0a0",
          "libstdcxx-ng >=10.3.0",
          "libtiff >=4.3.0,<5.0a0",
          "libzlib >=1.2.11,<1.3.0a0",
          "nss >=3.77,<4.0a0",
          "openjpeg >=2.4.0,<2.5.0a0",
          "poppler-data"
        ],
        "license": "GPL-2.0-only",
        "license_family": "GPL",
        "md5": "864687071a99df6f24dd587b77e4c753",
        "name": "poppler",
        "sha256": "53593388bb16933c2c711d0fb8e46b6cf6f429acb19a204b6e1cd6eaf1646288",
        "size": 20191738,
        "subdir": "linux-64",
        "timestamp": 1650986618544,
        "version": "22.04.0"
      }
    },
    {
      "op": "add",
      "path": "/packages/poppler-qt-22.04.0-hf358c63_0.tar.bz2",
      "value": {
        "build": "hf358c63_0",
        "build_number": 0,
        "depends": [
          "cairo >=1.16.0,<1.17.0a0",
          "fontconfig >=2.13.96,<3.0a0",
          "fonts-conda-ecosystem",
          "freetype >=2.10.4,<3.0a0",
          "gettext >=0.19.8.1,<1.0a0",
          "jpeg >=9e,<10a",
          "lcms2 >=2.12,<3.0a0",
          "libcurl >=7.82.0,<8.0a0",
          "libgcc-ng >=10.3.0",
          "libglib >=2.70.2,<3.0a0",
          "libiconv >=1.16,<1.17.0a0",
          "libpng >=1.6.37,<1.7.0a0",
          "libstdcxx-ng >=10.3.0",
          "libtiff >=4.3.0,<5.0a0",
          "libzlib >=1.2.11,<1.3.0a0",
          "nss >=3.77,<4.0a0",
          "openjpeg >=2.4.0,<2.5.0a0",
          "poppler 22.04.0 h1434ded_0",
          "qt >=5.12.9,<5.13.0a0"
        ],
        "license": "GPL-2.0-only",
        "license_family": "GPL",
        "md5": "33b3a474e4ecc5ab4e21c2f7912caf3d",
        "name": "poppler-qt",
        "sha256": "aa2c23a362c13837edf0763c77c98c97a3412640339924fa17c24f577d63b0ec",
        "size": 5553975,
        "subdir": "linux-64",
        "timestamp": 1650986746308,
        "version": "22.04.0"
      }
    }
  ]
}
{
  "url": "https://conda.anaconda.org/conda-forge/linux-64/repodata.json",
  "latest": "ea8a695e21baa8090691656bbe4a4d2240ee62cdc2539dc5940077ccf2a44efc",
  "headers": {
    "date": "Tue, 26 Apr 2022 18:14:12 GMT",
    "content-type": "application/json",
    "transfer-encoding": "chunked",
    "connection": "keep-alive",
    "cf-ray": "70214bf9bd722293-MIA",
    "age": "670",
    "cache-control": "public, max-age=1200",
    "etag": "W/\"30e1be6bf1b73889a41eaa4846a59c56-23\"",
    "expires": "Tue, 26 Apr 2022 18:34:12 GMT",
    "last-modified": "Tue, 26 Apr 2022 17:38:23 GMT",
    "vary": "Accept-Encoding",
    "cf-cache-status": "HIT",
    "expect-ct": "max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"",
    "x-amz-id-2": "G2rn+4SD+sHXNxisSHtEbxHLulnvMh/Bj+KotOHzI0zxZDval2V8My839uHEfIC7LjtQRcnTOFA=",
    "x-amz-request-id": "7X74ZXZB2DMZA92F",
    "x-amz-version-id": "null",
    "set-cookie": "__cf_bm=.XMuRKJKwivDP9dELOOpA8NcS__lncLVi06qA6hKELM-1650996852-0-AaceO99Yc9N2lSPDz67hzvxp+BIrIYJE9Wj7HrfdzxL1xiTCx3fG6Cj/d0kODkOelFlXUp+eRnJ0uZE/oLxFysUZVVIu4GUx7gtisLdg5L6i; path=/; expires=Tue, 26-Apr-22 18:44:12 GMT; domain=.anaconda.org; HttpOnly; Secure; SameSite=None",
    "server": "cloudflare",
    "content-encoding": "gzip"
  }
}
```